api_docs/changelog.md

```diff
@@ -18,6 +18,11 @@ clients should check the `zulip_feature_level` field, present in the
 /register`](/api/register-queue) responses, to determine the API
 format used by the Zulip server that they are interacting with.
 
+## Changes in Zulip 11.0
+
+Feature levels 372-375 reserved for future use in 10.x maintenance
+releases.
```

Comment:
You didn’t actually edit `API_FEATURE_LEVEL` for this (or forgot to `git add` or something).

Reply:
Not required until the next API change.